### ISSUE TITLE
Emails with Embedded Images are Only Showing Alt Text

### ISSUE THREAD

**Issue by Grunet** [Grunet/digestible-wcag#3]
The email for 1.4.11 (Non-Text Contrast) is one example, but there might be a few others out there as well

![image](https://user-images.githubusercontent.com/29322882/94760197-74bd2600-0367-11eb-8a95-6edb4bac139e.png)

[Link to view the email in a browser](https://htmlpreview.github.io/?https://raw.githubusercontent.com/Grunet/digestible-wcag-sc-emails/master/dist/1-4-11.html)

**Comment by Grunet** [comment]
The problem is that the "src" attributes of the images are pointing to relative paths, which makes sense when the images are viewed from within its WCAG page, but not on their own in the email's html

![image](https://user-images.githubusercontent.com/29322882/94760775-0c6f4400-0369-11eb-9305-9a9d4658ecf8.png)

```html
<img src="img/graphics-contrast_pie-chart_fail.png" alt="Failing pie chart">
```
Gmail specifically is also modifying the "src" attributes so that the images are fetched behind it's proxy, but I'm guessing that's not really part of the problem here.

![image](https://user-images.githubusercontent.com/29322882/94760712-e8abfe00-0368-11eb-9807-43f47cb598a6.png)

```html
<img src="https://ci3.googleusercontent.com/proxy/kcm6jB5sZDvWy70pypHy5FZOTmLxiCkmO1ENk041rkXpMa9k2oEKUjEaoadxa-Rs5I9fbRiFVjSfN4VhReMhhZ7G=s0-d-e1-ft#http://img/graphics-contrast_pie-chart_fail.png" alt="Failing pie chart" class="CToWUd a6T" tabindex="0">
```

**Comment by Grunet** [comment]
This should be fixable by adding in a format.ts file into the /src/understandingPages/ directory of the digestible-wcag-scraping repository (similar to the one underneath the /src/mainDocument/ directory) that scans the examples' html for any "img" tags that have relative paths, and then sticks on the base of the path to it, ending up with something like this (the same thing you get if you right-click to do "View Image" when you're on its Understanding WCAG page itself)

```
https://www.w3.org/WAI/WCAG21/Understanding/img/graphics-contrast_pie-chart_fail.png
```

And then those changes would need to be propagated into the wcag22.json output as well as the generated emails themselves in the digestible-wcag-sc-emails repository by running the appropriate VS Code tasks

**Comment by Grunet** [comment]
This also affects [the email for 1.4.13 (Content on Hover or Focus)](https://htmlpreview.github.io/?https://raw.githubusercontent.com/Grunet/digestible-wcag-sc-emails/master/dist/1-4-13.html).

**Comment by Grunet** [comment]
Just performed the fix and verified that the images were showing up locally (from VS Code's HTML Preview) in the 2 emails
